cdi/pom.xml

```diff
@@ -88,7 +88,6 @@
         <dependency>
             <groupId>javax.enterprise</groupId>
             <artifactId>cdi-api</artifactId>
-            <scope>provided</scope>
         </dependency>
```

Comment:
The quickstart doesn't build against the latest infinispan SNAPSHOT on the jdg-6.6.x branch (which will be part of 6.6.0.DR2) without this dependency.